<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectDialog</class>
 <widget class="QDialog" name="ConnectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>253</width>
    <height>242</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connection</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>171</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="socketTab">
    <attribute name="title">
     <string>TCP/IP Socket</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>196</width>
       <height>94</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="1,0">
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="2" column="0">
       <widget class="QLabel" name="addressLabel">
        <property name="text">
         <string>Address:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="portLabel">
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="portSpinBox">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>4000</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QComboBox" name="addressComboBox">
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::InsertAtTop</enum>
        </property>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="usernameEdit"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="usernameLabel">
        <property name="text">
         <string>Username:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="webCheckBox">
        <property name="text">
         <string>Use web server</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="ircTab">
    <attribute name="title">
     <string>IRC</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>196</width>
       <height>146</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,0">
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="4" column="0">
       <widget class="QLabel" name="addressLabel_2">
        <property name="text">
         <string>IRC Server</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="portLabel_2">
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="portSpinBox_2">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>6667</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QComboBox" name="ircServerEdit">
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::InsertAtTop</enum>
        </property>
        <item>
         <property name="text">
          <string>irc.austnet.org</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>127.0.0.1</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="ircNickEdit"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="usernameLabel_2">
        <property name="text">
         <string>IRC Nick</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="gmNickEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="usernameLabel_3">
        <property name="text">
         <string>GM Nick</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QComboBox" name="ircChannelEdit">
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::InsertAtTop</enum>
        </property>
        <item>
         <property name="text">
          <string>#caIRCtest</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="addressLabel_3">
        <property name="text">
         <string>Data Channel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QCheckBox" name="showStartupCheckBox">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>215</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Show connect dialog at startup</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connectButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="listenButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Listen</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>usernameEdit</tabstop>
  <tabstop>addressComboBox</tabstop>
  <tabstop>portSpinBox</tabstop>
  <tabstop>showStartupCheckBox</tabstop>
  <tabstop>connectButton</tabstop>
  <tabstop>listenButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
